<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_ledOn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LED ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ledOff">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LED OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buzzerOn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>buzzer ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buzzerOff">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>buzzer OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sonicOn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>UltraSonic ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sonicOff">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>UltraSonic OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_sonic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
